.gitignore

```diff
@@ -6,3 +6,4 @@ yarn-error.log
 .build
 __docs__
```

Comment:
maybe `__docs__` can be removed from here now? not sure if we still use it

Reply:
We still use it for generated API docs.